allennlp/modules/token_embedders/embedding.py

```diff
@@ -140,6 +149,25 @@ def forward(self, inputs):  # pylint: disable=arguments-differ
             embedded = projection(embedded)
         return embedded
 
+    @overrides
+    def extend_vocab(self, extended_vocab: Vocabulary, vocab_namespace: Optional[str] = None):
+        if not vocab_namespace:
+            vocab_namespace = getattr(self, "_vocab_namespace", None)
+
+        if not vocab_namespace:
+            vocab_namespace = "tokens"
+            logging.warning("Neither original training store vocab_namespace for Embedding class "
```

Comment:
Wording tweak: just say "No vocab_namespace provided to Embedder.extend_vocab. Defaulting to 'tokens'."

Reply:
Sure 👍